packages/clay-form/src/Input.tsx

```diff
@@ -198,6 +199,7 @@ const ClayInput = React.forwardRef<HTMLInputElement, IProps>(
 				['input-group-inset-before']: insetBefore,
 			})}
 			ref={ref}
+			type={Component === 'input' ? type : undefined}
```

Comment:
A little strange, why not just pass the `type`? it seems safe to just pass the type attribute.

Reply:
It was a leftover from textarea commits, since a textarea doesn't have `type` attr. I'll remove.